tests/foreman/cli/test_classparameters.py

```diff
@@ -31,68 +31,54 @@
 from robottelo.config import settings
 from robottelo.constants.repos import CUSTOM_PUPPET_REPO
 from robottelo.datafactory import gen_string
-from robottelo.test import CLITestCase
 
 
-@pytest.mark.run_in_one_thread
-class SmartClassParametersTestCase(CLITestCase):
-    """Implements Smart Class Parameter tests in CLI"""
+@pytest.fixture(scope='module')
+def module_org():
```

Comment:
this name is used at the global level in API fixtures. if your setup doesn't require the org be created via cli, please use that instead. Otherwise, it would be more clear to change this name to something else like `cli_module_org`.

Reply:
ack